notebooks/community/model_garden/model_garden_pytorch_mistral_peft_tuning.ipynb

```diff
@@ -0,0 +1,1460 @@
+{
```

Comment:
<p>Update the license year to 2024.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2645/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>